autoscheduler/frontend/src/.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'airbnb',
```

Comment:
On the tutorial I followed, it also had 
`        'plugin:react/recommended', `
`        'plugin:@typescript-eslint/recommended', `

As an extended plugin. Is this something we need to add?

Reply:
Ah, yes, thank you. These two plugins overwrite several ESLint rules that conflict with TypeScript and React development, so I added them in